### DISCUSSION TITLE
Remove snowflake border

### DISCUSSION THREAD

**Discussion by joetech** [photobooth-app/photobooth-app#158] [Q&A] [answered]
I'm using the app to give me a live preview for getting my camera set up how I need it before loading up AllSky.  It all works well enough, but when I take a photo, I get this snowflake border.  I can't find any reference to it or how to remove it.

![image](https://github.com/photobooth-app/photobooth-app/assets/1650878/4cf99a7b-4af0-48ea-b9e8-d748b79b46e8)

**Comment by mgineer85** [comment, marked as answer]
Hehe, the default setup is made to demo the postprocessing a bit.
Just switch off all stages in the section mediaprocessing in the admin dashboard config.

**Reply by mgineer85** [reply]
https://photobooth-app.org/setup/mediaprocessing/#picture-pipeline

Here it’s described, but could be improved probably.

**Reply by mgineer85** [reply]
Also Maybe once its setup you want to use the rest api.
that gives you direct access to the captured image as is from the camera, see https://photobooth-app.org/reference/api/
section /aquisition/still
